interface/src/app/gene/network/network.tpl.html

```diff
@@ -14,25 +14,37 @@
   <h3 class="text-warning" ng-bind="ctrl.statusMessage"></h3>
 
   <div ng-hide="ctrl.statusMessage">
+    <div class="well well-sm">
     <form>
-      <strong>Correlation Range:</strong>
+      <strong>Min edge weight:</strong>
       <label>
         <input type="checkbox" ng-model="ctrl.includeNegative"
                ng-disabled="!ctrl.includePositive"
                ng-change="ctrl.renderNetwork()">
-        Include negative correlation
+        Include negative weight
       </label>
       &nbsp;
       <label>
         <input type="checkbox" ng-model="ctrl.includePositive"
                ng-disabled="!ctrl.includeNegative"
                ng-change="ctrl.renderNetwork()">
-        Include positive correlation
+        Include positive weight
       </label>
     </form>
-    <rzslider rz-slider-model="ctrl.slider.min"
-              rz-slider-options="ctrl.slider.options">
+    <rzslider id="edge-slider"
+              rz-slider-model="ctrl.minEdgeWeightSlider.value"
+              rz-slider-options="ctrl.minEdgeWeightSlider.options">
     </rzslider>
+    </div>
+
+    <div class="well well-sm">
+      <strong>Max # of genes:</strong>
```

Comment:
I think it would be helpful to include some information for the user about *how* the genes are filtered. We are using some kind of rank, right? It is not clear from the UI how changing this slider alters which genes are shown and which are hidden.

Reply:
It is not easy to explain the exact algorithm that ranks the genes. I had to walk through the code in `d3.network.js` with Casey to fully understand it. I think this kind of information should be added in `README.md` or the tech help page.

Reply:
I was wondering about this as well. I assumed the same thing, but it might be good to clarify to the users.

Reply:
I agree, if it's not that trivial then clarifying in the README might make more sense

Reply:
This ties in with the ranking code where I'm asking for more detail. I think we need a clear picture of how this works. If we cannot explain it, we do not understand it ourselves and I think that is a problem.